opencti-platform/opencti-graphql/src/modules/playbook/playbook-components.ts

```diff
@@ -566,13 +566,11 @@ const attributePathMapping: any = {
     [ABSTRACT_STIX_RELATIONSHIP]: `/${objectMarking.stixName}`,
   },
   [INPUT_LABELS]: {
-    [ABSTRACT_STIX_DOMAIN_OBJECT]: `/${objectLabel.stixName}`,
-    [ABSTRACT_STIX_CYBER_OBSERVABLE]: `/extensions/${STIX_EXT_OCTI_SCO}/${objectLabel.stixName}`,
```

Comment:
Could you explain the change please?

Reply:
The PLAYBOOK_UPDATE_KNOWLEDGE_COMPONENT that manipulates knowledge uses those attributePathMapping to decide where to put the added labels configured in the playbook. But it was using an incorrect path to put those labels: it was putting them in the /extensions/${STIX_EXT_OCTI_SCO} path of the bundle, instead of putting them at the root of the stix object. Because they were at the incorrect path, they were ignored during the bundle ingestion by the worker